include/wx/flags.h

```diff
@@ -155,7 +155,7 @@ private :
 #if wxUSE_EXTENDED_RTTI
 
 #define wxDEFINE_FLAGS( flags ) \
-    class WXDLLIMPEXP_BASE flags \
+    class WXDLLIMPEXP_CORE flags \
```

Comment:
I think there should be no DLL export declaration here at all -- it's useless for classes implemented fully inline (and sometimes can be harmful with MSVC).

Reply:
If XTI will be removed, this is indeed not necessary (`wxDEFINE_FLAGS` will be removed entirely then).